services/graph/pkg/service/v0/driveitems.go

```diff
@@ -472,142 +470,92 @@ func (g Graph) Invite(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	createShareErrors := sync.Map{}
-	createShareSuccesses := sync.Map{}
+	driveRecipient := driveItemInvite.GetRecipients()[0]
```

Comment:
shouldn't we check the length of `driveItemInvite.GetRecipients()`? if we just silently pick only the first all the others will silently be ignored?

Reply:
It’s covered in the validator to have a strict Len of 1